tests/dir.sdkbuild.props

```diff
@@ -0,0 +1,25 @@
+<Project>
+  <Import Project="../dir.common.props" />
+
+  <!-- This file contains build properties that apply to every SDK
+       project under the test tree, and also generated test projects
+       in bin/tests. Eventually every test project should build using
+       the SDK. -->
+
+  <PropertyGroup>
+    <OSPlatformConfig>$(BuildOS).$(Platform).$(Configuration)</OSPlatformConfig>
+
+    <TestSrcDir>$(CoreclrDir)/tests/src</TestSrcDir>
```

Comment:
I like how many of these paths are being unified for testing, but it seems odd that `TestSrcDir` and `TestWorkingDir` aren't defined together.

Reply:
It looks like `TestWorkingDir` is set in coreclr/dir.props, and used by the helix projects and runtest.cmd. For this change, I only factored out the properties used by SDK-style projects. None of the stuff in coreclr/dir.props is used by the new projects. Perhaps @chsienki's work will move more (everything?) over to a single props file.